mozphab/jujutsu.py

```diff
@@ -36,7 +37,23 @@ class Jujutsu(Repository):
     def __init__(self, path: str):
         # NOTE: We expect a co-located Git repository alongside the Jujutsu repository, so let's
         # start up a Git backend to handle the pieces we need.
-        self.__git_repo = Git(path)
+
+        # Since there could be multiple Jujutsu workspaces, and the underlying
+        # Git repository could be in either one of them, or in another folder
+        # entirely, we will need to do a small search first.
+        self.git_path = Path(path)
+        repo = self.git_path / ".jj" / "repo"
+        if repo.is_file():
+            self.git_path = Path(repo.read_text()).parent.parent
+        git_target = self.git_path / ".jj" / "repo" / "store" / "git_target"
+        if git_target.is_file():
+            self.git_path = Path(git_target.read_text()).parent
```

Comment:
This logic didn't actually seem to work…so I pushed up a fixed version with 30d145e, which has steps to break down individual path manipulations necessary to get the correct path. How's it look?